sebs/azure/blob_storage.py

```
@@ -26,16 +29,32 @@ def __init__(
+        :param conn_string: Azure Storage connection string.
+        :param replace_existing: Flag to replace existing files in containers.
+        """
         super().__init__(region, cache_client, resources, replace_existing)
         self.client: BlobServiceClient = BlobServiceClient.from_connection_string(conn_string)
 
-    """
-        Internal implementation of creating a new container.
-    """
-
     def _create_bucket(
         self, name: str, containers: List[str] = [], randomize_name: bool = False
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)